Makefile

```diff
@@ -26,8 +26,27 @@ PACKAGE_LIST  := go list ./...
 PACKAGES  := $$($(PACKAGE_LIST))
 FAIL_ON_STDOUT := awk '{ print } END { if (NR > 0) { exit 1 } }'
 
+FAILPOINT_DIR := $$(for p in $(PACKAGES); do echo $${p\#"github.com/pingcap/tidb-tools/"}; done)
+FAILPOINT := retool do failpoint-ctl
+FAILPOINT_ENABLE  := $$(echo $(FAILPOINT_DIR) | xargs $(FAILPOINT) enable >/dev/null)
+FAILPOINT_DISABLE := $$(find $(FAILPOINT_DIR) | xargs $(FAILPOINT) disable >/dev/null)
+
+define run_unit_test
+	@echo "running unit test for packages:" $(1)
+	$(FAILPOINT_ENABLE)
+	@export log_level=error; \
+	$(GOTEST) -cover $(PACKAGES) \
```

Comment:
Can we use `$(1)` to replace `$(PACKAGES)`?

Reply:
updated